src/robusta/core/triggers/pod_oom_killed_trigger.py

```diff
@@ -0,0 +1,74 @@
+from ...core.playbooks.base_trigger import TriggerEvent
+from ...integrations.kubernetes.autogenerated.triggers import (
+    PodUpdateTrigger,
+    PodChangeEvent,
+)
+from ...integrations.kubernetes.base_triggers import K8sTriggerEvent
+from ...utils.rate_limiter import RateLimiter
+
+
+class PodOOMKilledTrigger(PodUpdateTrigger):
+    """
+    :var rate_limit: Limit firing to once every `rate_limit` seconds
+    """
+
+    rate_limit: int = 3600
```

Comment:
Maybe on oom kills, the default should be to report on all oom killed.
So maybe the rate limit should be defaulted to 0